webapp/src/pages/UserStatsPage/UserStatsPage.css

```diff
@@ -0,0 +1,49 @@
+.stats-page {
+  height: 100vh;
+  padding: var(--padding-top-bottom) var(--padding-sides);
+  display: grid;
+  grid-template-columns: repeat(var(--column-number), 1fr);
+  grid-template-rows: auto 1fr;
+  gap: 3rem var(--column-gap);
+  background-color: var(--color-background);
+  color: var(--color-light);
+  max-height: available;
+  overflow: clip;
+}
+
+.stats-page .header {
+  grid-column: 1/-1;
+  grid-row: 1/2;
+  align-self: center;
+}
+
+.stats-page .graphs {
+  grid-column: 1/-1;
+  grid-row: 2/-1;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  gap: var(--row-gap) var(--column-gap);
+}
+
+.stats-page .graphs .line-graph-wrapper {
+  display: flex;
+  justify-content: center;
+}
+
+.stats-page .graphs .line-graph-wrapper canvas {
+  min-height: 18rem;
```

Comment:
the style could maybe use some more love, but it looks good, it's boring and I'm tired, so this is it.